setup.sh

```diff
@@ -56,9 +57,6 @@ cd ../libpcap
 make $MAKE_FLAGS
 make install
 
-#Make PF Ring's libpcap available to the compiler
-echo "/usr/local/pfring/lib" >> /etc/ld.so.conf
```

Comment:
I took this out and Bro still linked against the PF_RING libpcap. I didn't think it was necessary since the path is specified explicitly in the `./configure` command. If there are other reasons this was included then I can put it back in.

Reply:
This was the result of paranoia / wanting to make sure other apps got pfring.